src/calibrationlib/csv_writer.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "boost/bimap.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+class CSVWriter {
+ public:
+  CSVWriter(std::string file_path, const std::vector<std::string> headers, const char delimiter = ',',
```

Comment:
Not a strong opinion on that, but does it make sense to be able to change the delimiter here? I would expect the delimiter to stay the same for a complete "calibration project". Therefore, I could imagine having delimiter as a constant.